notebook/templates/page.html

```diff
@@ -118,7 +118,7 @@
 
 <div id="header">
   <div id="header-container" class="container">
-  <div id="ipython_notebook" class="nav navbar-brand pull-left"><a href="{{default_url}}" title='{% trans %}dashboard{% endtrans %}'>{% block logo %}<img src='{{static_url("base/images/logo.png") }}' alt='Jupyter Notebook'/>{% endblock %}</a></div>
+  <div id="ipython_notebook" class="nav navbar-brand pull-left"><a href="{{default_url}}?token={{token}}" title='{% trans %}dashboard{% endtrans %}'>{% block logo %}<img src='{{static_url("base/images/logo.png") }}' alt='Jupyter Notebook'/>{% endblock %}</a></div>
```

Comment:
This exposes the login token on the login page, so unauthenticated users can just click this link to login. We should make sure to only include this if the page being served is already authenticated, and only when the token is defined.